iothub/service/src/Messaging/Models/PurgeMessageQueueResult.cs

```diff
@@ -23,13 +23,13 @@ protected internal PurgeMessageQueueResult()
         /// <summary>
         /// The Id of the device whose messages are being purged.
         /// </summary>
-        [JsonProperty("deviceId", Required = Required.Always)]
+        [JsonProperty("deviceId")]
```

Comment:
We don't need to validate items are in the payload when de/serializing. We can trust the service included it, or if the user didn't include a required item, the service should return an error.